test/support/test_user.ex

```diff
@@ -5,15 +5,17 @@ defmodule Ecto.Ldap.TestUser do
   @primary_key {:dn, :string, autogenerate: false}
   schema "users" do
     field :objectClass, :string
+    field :languages, {:array, :string}
     field :mail, :string
     field :mobile, :string
+    field :skills, {:array, :string}
```

Comment:
Added skills and languages to our test model for testing purposes (imagine that!!)